### DISCUSSION TITLE
Allow to select default view for Calendar Dashboard

### DISCUSSION THREAD

**Discussion by cadavre** [home-assistant/frontend#7632] [Dashboard]
Currently monthly view is default display for Calendar dashboard. In cards now we can select `initial_view`, but this option is still not there for calendar dashboard view.

Maybe adding a configuration for Calendar dashboard would work?

**Comment by kristianfreeman** [comment]
Woah, old thread! I would be interested in this as well. Is there any interest from the team in a submission that would persist initial_view into local storage?
